qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -98,6 +98,14 @@
                             cpu_model = host
                         - default_model:
                             cpu_model_flags += ",+pcid,+invpcid"
+                - v_spec_ctrl:
+                    flags = "v_spec_ctrl"
+                    type = x86_cpu_v_spec_ctrl
+                    supported_models = "EPYC-Milan"
+                    boot_option = "spec_store_bypass_disable=on"
+                    test_dir = '/root'
```

Comment:
I don't rem clearly. Do we have a rule for the test directory? @yanan-fu Could you please help clarify? Thanks.

Reply:
> I don't rem clearly. Do we have a rule for the test directory? @yanan-fu Could you please help clarify? Thanks.

Confirmed with yfu, the specific test directory is not a must as long as we clear the test env finally.
And change to the recommended directory, thanks.